aiohttp/web_response.py

```diff
@@ -42,6 +42,12 @@
     BaseClass = collections.abc.MutableMapping
 
 
+if 'samesite' not in Morsel._reserved:  # type: ignore
```

Comment:
Is `helpers.PY_38` check here instead if `Morsel._reserved` lookup.
It makes the intention explicitly clear and allows dropping the patch after dropping Python 3.7 support.